eZ Platform/Admin UI/Field Type Edit/Requirements.md

```diff
@@ -0,0 +1,45 @@
+# Content editing
+
+
+# Objective
+
+"As an eZ Engineer, I want...
+
+To create a functional, experimental implementation of a refactoring of the Field Type API in eZ Platform supporting full CRUD (create, read, update and delete) for content creation and editing. The purpose of the research is to find a common way to define the user interface using different technologies."
+
+Underlying goals are:
+
+* Simplify how to create new custom Field Type
+* Reduce errors by limiting the number of places UI definitions are done
+* Increase developer velocity when developing and maintaining Field Types
+
+# Firm
+
+The prototype *must* for the following interfaces:
+* Twig based content editing
+* A remote website (e.g. Angular)
```

Comment:
@rolandbenedetti Is Twig a must for editing? Or was it meant to say Symfony? Cause if we want to find common ground we should avoid defining view technology in the must section.